### PR TITLE
feat: support to cosmwasm 1.1

### PR DESCRIPTION
**Pull request by JoowonYun**
- feat: support to cosmwasm 1.1
- refactor: remove third_party proto

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -207,6 +208,7 @@ var (
 		router.AppModuleBasic{},
 		ica.AppModuleBasic{},
 		wasm.AppModuleBasic{},
+		ica.AppModuleBasic{},
```
Comment (yoosah) [inline app/app.go:211]: dup

File: app/app.go [hunk 2]
```diff
@@ -354,7 +358,7 @@ func NewXplaApp(
 	scopedIBCKeeper := app.CapabilityKeeper.ScopeToModule(ibchost.ModuleName)
 	scopedTransferKeeper := app.CapabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
 	scopedICAHostKeeper := app.CapabilityKeeper.ScopeToModule(icahosttypes.SubModuleName)
-
+	// this line is used by starport scaffolding # stargate/app/scopedKeeper
```
Comment (yoosah) [inline app/app.go:361]: redundant?

**Review by yoosah — COMMENTED** [review]
check comments please

File: app/app.go [hunk 3]
```diff
@@ -728,6 +748,30 @@ func NewXplaApp(
...
+		slashing.NewAppModule(appCodec, app.SlashingKeeper, app.AccountKeeper, app.BankKeeper, app.StakingKeeper),
+		params.NewAppModule(app.ParamsKeeper),
+		evidence.NewAppModule(app.EvidenceKeeper),
+		wasm.NewAppModule(appCodec, &app.wasmKeeper, app.StakingKeeper, app.AccountKeeper, app.BankKeeper),
+		ibc.NewAppModule(app.IBCKeeper),
+		transferModule,
+	)
+
+	app.sm.RegisterStoreDecoders()
+
```
Comment (yoosah) [inline app/app.go:774]: where do you use simulation manager?

**Review by yoosah — COMMENTED** [review]
check comments please

File: app/app.go (hunk 1 above)
Comment (JoowonYun) [reply app/app.go]: https://github.com/xpladev/xpla/pull/39/commits/958ab950e6cb3c7d1d1b184f232ca51a71d7d55e

**Comment by JoowonYun** [comment]
https://github.com/xpladev/xpla/pull/85

**Pull request closed without merging**